.github/CODEOWNERS

```diff
@@ -28,7 +28,7 @@
 # Todo: is this file still needed?
 /Makefile.trace                         @DataDog/agent-platform
 
-/release.json                           @DataDog/agent-platform @DataDog/agent-metrics-logs @DataDog/windows-kernel-integrations
```

Comment:
Do you think we could add `@DataDog/agent-security` as well (for the security agent policies tag)

Reply:
Sure. Adding.

Reply:
Thank you !